mkdocs.yml

```diff
@@ -9,4 +9,4 @@ pages:
     - Block Reader: operators/block_reader.md
     - File Output: operators/file_output.md
     - Enricher: operators/enricher.md
-    - Deduper: operators/deduper.md
+    - Json Parser: operators/jsonParser.md
```

Comment:
This is going to conflict with your other review of jsonformatter. So we'll have to merge this and json formatter one by one and rebase will be required.